src/api/firebase.js

```diff
@@ -11,7 +11,7 @@ import {
 } from 'firebase/firestore';
 import { useEffect, useState } from 'react';
 import { db } from './config';
-import { getFutureDate } from '../utils';
+import { getFutureDate, getDaysBetweenDates } from '../utils';
```

Comment:
This isn't being used, and looks to be superseded by `getDaysBetweenLastAndNextPurchase` in the ListItem component. However, we want to use the logic in the dates file as that is a good, pure function that keeps the logic out of the component code.

Reply:
See req:  

> A getDaysBetweenDates function is exported from utils/dates.js and imported into api/firebase.js

> This function takes two JavaScript Dates and return the number of days that have passed between them